commit b884ba5222a765b395e8ac93971639a0452d6422
Author: Dirkjan Bussink <d.bussink@gmail.com>
Date:   Fri Sep 14 14:31:22 2018 +0200

    CLEANUP: stick-tables: Remove unneeded double (()) around conditional clause
    
    In the past this conditional had multiple conditionals which is why the
    additional parentheses were needed. The conditional was simplified but
    the duplicate parentheses were not cleaned up.
    
    (cherry picked from commit ff57f1bbcf8af1e6389520aa845df5aa97ef55b6)
    [wt: fixes build warnings with clang]
    Signed-off-by: Willy Tarreau <w@1wt.eu>

diff --git a/src/stick_table.c b/src/stick_table.c
index 653a1ffb..f1442603 100644
--- a/src/stick_table.c
+++ b/src/stick_table.c
@@ -1860,7 +1860,7 @@ smp_fetch_sc_tracked(const struct arg *args, struct sample *smp, const char *kw,
 	smp->data.u.sint = !!stkctr;
 
 	/* release the ref count */
-	if ((stkctr == &tmpstkctr))
+	if (stkctr == &tmpstkctr)
 		stktable_release(stkctr->table, stkctr_entry(stkctr));
 
 	return 1;
